src/main/resources/application.properties

```diff
@@ -65,4 +65,26 @@ redis.insights.index=12
 learner.service.url=http://learnerServiceIP:9010
 sb.org.search.path=/v1/org/search
 sb.api.key=apikey
-bookmark.duplicate.not.allowed.category=MDO_ORG_LIST
\ No newline at end of file
+bookmark.duplicate.not.allowed.category=MDO_ORG_LIST
+
+#kafka Server
+spring.kafka.bootstrap.servers=localhost:9092
+#kafka properties
+kakfa.offset.reset.value=latest
+kafka.max.poll.interval.ms=15000
+kafka.max.poll.records=100
+kafka.auto.commit.interval.ms=10000
+#kafka topics and group
+kafka.topic.demand.request=dev.demand.request
+kafka.topic.demand.content.group="demand.request.group"
+
+#notification-Service
+notify.service.host=http://localhost:6006
```

Comment:
provide the address which is used by kong mapping file for notification service url